flink-walkthroughs/flink-walkthrough-datastream-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -31,7 +31,6 @@ under the License.
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<flink.version>@project.version@</flink.version>
-		<target.java.version>1.8</target.java.version>
```

Comment:
We should not remove this, but change it to `@target.java.version@` I think. The built archetype will be independent from the flink project when it is installed, so once it built it should have a raw version inside that prop, but it should be resolved from the root POM version during build time, just like it happens already for `flink.version` and `log4j.version` props.

Sorry if my previous comment was misleading. :)

Reply:
no worries -  sorry misunderstood - I will reinstate this.